src/services/API/Video.js

```diff
@@ -0,0 +1,20 @@
+import { apiClient } from "@/services/API/RootClient.js";
+
+export default {
+  async createVideo(videoDetails) {
+    // API to create a video object in the DB
+    var response = await apiClient().post(
+      process.env.VUE_APP_BACKEND_VIDEOS,
+      videoDetails
+    );
+    return response.data.id;
```

Comment:
any reason for returning only the id and not the whole object?

I think these service files should return the response we're getting and let the caller function decide what data it needs. That will help making these service functions more generic.

Reply:
Agreed. This was specific to my use case. Will update it!